<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>742</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>7360</width>
    <height>6460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Master Tile Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/data/icon</normaloff>:/data/icon</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <layout class="QGridLayout" name="main_gridLayout" rowstretch="0,0,0" columnstretch="0,0">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Zoom</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="TilesWidget" name="tileswidget" native="true">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="tilesScrollBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tiles</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Palette</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Use palette:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="Apply_offset_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PaletteWidget" name="palettewidget" native="true">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="offset_lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Offset of the first pixel displayed</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Palette Part:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="background_palette_radioButton">
           <property name="text">
            <string>Background</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="sprite_palette_radioButton">
           <property name="text">
            <string>Sprite</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="palette_pushButton">
          <property name="text">
           <string>Palette at offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bookmark_pushButton">
          <property name="text">
           <string>Bookmark offset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current offset: 0x</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QComboBox" name="palette_comboBox"/>
      </item>
      <item row="3" column="2" colspan="4">
       <widget class="QComboBox" name="bookmark_comboBox"/>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Go to bookmark:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="ZoomWidget" name="zoomwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="tile_offset_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="5" column="1">
         <widget class="QPushButton" name="down_1tile_pushButton">
          <property name="text">
           <string>1 tile</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="up_160tiles_pushButton">
          <property name="text">
           <string>96 tiles</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="down_160tiles_pushButton">
          <property name="text">
           <string>96 tiles</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="up_16tiles_pushButton">
          <property name="text">
           <string>16 tiles</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="down_1row_pushButton">
          <property name="text">
           <string>1 row</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="down_16tiles_pushButton">
          <property name="text">
           <string>16 tiles</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="up_1row_pushButton">
          <property name="text">
           <string>1 row</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Down</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="down_1Byte_pushButton">
          <property name="text">
           <string>1 byte</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="up_1Byte_pushButton">
          <property name="text">
           <string>1 byte</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Up</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="up_1tile_pushButton">
          <property name="text">
           <string>1 tile</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Move:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="return_realrom_pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Return to ROM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="find_next_compr_pushButton">
        <property name="text">
         <string>Find next
compressed tiles</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Mode:</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="1" colspan="2">
          <widget class="QRadioButton" name="mode_3bpp_radioButton">
           <property name="text">
            <string>3 bpp</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="mode_1bpp_radioButton">
           <property name="text">
            <string>1 bpp</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="mode_4bpp_radioButton">
           <property name="text">
            <string>4 bpp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="mode_2bpp_radioButton">
           <property name="text">
            <string>2 bpp</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>ROM</string>
    </property>
    <addaction name="actionOpen_Rom"/>
    <addaction name="actionSave_Rom"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_IPS_patch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHacks">
    <property name="title">
     <string>Hacks</string>
    </property>
    <addaction name="actionApply_Hack_File_fast"/>
    <addaction name="actionApply_Hack_File_confirm"/>
   </widget>
   <widget class="QMenu" name="menuCompressed_Tiles">
    <property name="title">
     <string>Compression</string>
    </property>
    <addaction name="actionDecompress_Tiles"/>
    <addaction name="separator"/>
    <addaction name="actionCompress_Picture"/>
    <addaction name="actionImport_Compressed_Data"/>
    <addaction name="actionCompressed_and_Import_Data"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Palette"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Picture"/>
    <addaction name="actionImport_Picture"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAssembly">
    <property name="title">
     <string>Assembly</string>
    </property>
    <addaction name="actionTiles2asm"/>
    <addaction name="actionPalette2asm"/>
    <addaction name="actionImage2asm"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuCompressed_Tiles"/>
   <addaction name="menuHacks"/>
   <addaction name="menuAssembly"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Rom">
   <property name="text">
    <string>Open ROM</string>
   </property>
  </action>
  <action name="actionImport_Picture">
   <property name="text">
    <string>Import Picture</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport_Palette">
   <property name="text">
    <string>Import Palette</string>
   </property>
  </action>
  <action name="actionExport_Picture">
   <property name="text">
    <string>Export Picture</string>
   </property>
  </action>
  <action name="actionSave_Rom">
   <property name="text">
    <string>Save ROM</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionImport_Compressed_Data">
   <property name="text">
    <string>Import Compressed Data</string>
   </property>
  </action>
  <action name="actionApply_Hack_File_fast">
   <property name="text">
    <string>Apply Hack File (fast)</string>
   </property>
  </action>
  <action name="actionApply_Hack_File_confirm">
   <property name="text">
    <string>Apply Hack (confirm)</string>
   </property>
  </action>
  <action name="actionCompress_Picture">
   <property name="text">
    <string>Compress Picture</string>
   </property>
  </action>
  <action name="actionDecompress_Tiles">
   <property name="text">
    <string>Find Next Compressed Tiles</string>
   </property>
  </action>
  <action name="actionCompressed_and_Import_Data">
   <property name="text">
    <string>Compress and Import Data</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionTiles2asm">
   <property name="text">
    <string>Tiles to Asm</string>
   </property>
  </action>
  <action name="actionPalette2asm">
   <property name="text">
    <string>Palette to Asm</string>
   </property>
  </action>
  <action name="actionImage2asm">
   <property name="text">
    <string>BMP to Asm</string>
   </property>
  </action>
  <action name="actionCreate_IPS_patch">
   <property name="text">
    <string>Create IPS patch</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TilesWidget</class>
   <extends>QWidget</extends>
   <header>tileswidget.h</header>
  </customwidget>
  <customwidget>
   <class>PaletteWidget</class>
   <extends>QWidget</extends>
   <header>palettewidget.h</header>
  </customwidget>
  <customwidget>
   <class>ZoomWidget</class>
   <extends>QWidget</extends>
   <header>zoomwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>up_1tile_pushButton</tabstop>
  <tabstop>down_1Byte_pushButton</tabstop>
  <tabstop>down_1tile_pushButton</tabstop>
  <tabstop>up_16tiles_pushButton</tabstop>
  <tabstop>down_16tiles_pushButton</tabstop>
  <tabstop>up_1row_pushButton</tabstop>
  <tabstop>down_1row_pushButton</tabstop>
  <tabstop>up_1Byte_pushButton</tabstop>
  <tabstop>down_160tiles_pushButton</tabstop>
  <tabstop>up_160tiles_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
